package.json

```diff
@@ -21,7 +21,8 @@
     "husky": "^9.0.11",
     "prettier": "^2.5.1",
     "pretty-quick": "^3.1.3",
-    "turbo": "^1.13.3"
+    "turbo": "^1.13.3",
+    "openai": "^4.1.0"
```

Comment:
since most folks won't need this, I think it would be better to make this an `optionalDependency`. Then you can install it at runtime via:

(example is chatGPT generated)

```javascript
const { execSync } = require('child_process');

function requireOrInstall(packageName) {
  try {
    // Try to require the package
    return require(packageName);
  } catch (e) {
    if (e.code === 'MODULE_NOT_FOUND') {
      // Package not found, install it
      console.log(`${packageName} not found. Installing...`);
      execSync(`npm install ${packageName}`, { stdio: 'inherit' });

      // Require the package again after installation
      return require(packageName);
    } else {
      throw e;
    }
  }
}

...

const { OpenAI } = requireOrInstall('openai');